ctru-rs/src/services/apt.rs

```diff
@@ -82,6 +82,48 @@ impl Apt {
             Ok(())
         }
     }
+
+    /// Set if the console is allowed to enter sleep mode.
+    ///
+    /// You can check whether the console is allowed to sleep with [Apt::is_sleep_allowed].
+    #[doc(alias = "aptSetSleepAllowed")]
+    pub fn set_sleep_allowed(&mut self, allowed: bool) {
+        unsafe {
+            ctru_sys::aptSetSleepAllowed(allowed);
+        }
+    }
+
+    /// Check if the console is allowed to enter sleep mode.
+    ///
+    /// You can set whether the console is allowed to sleep with [Apt::set_sleep_allowed].
+    #[doc(alias = "aptIsSleepAllowed")]
+    pub fn is_sleep_allowed(&self) -> bool {
+        unsafe { ctru_sys::aptIsSleepAllowed() }
+    }
+
+    /// Set if the console is allowed to enter the home menu.
+    ///
+    /// You can check whether the console is allowed to enter the home menu with [Apt::is_home_allowed].
+    #[doc(alias = "aptSetHomeAllowed")]
+    pub fn set_home_allowed(&mut self, allowed: bool) {
+        unsafe {
+            ctru_sys::aptSetHomeAllowed(allowed);
+        }
+    }
+
+    /// Check if the console is allowed to enter the home menu.
+    ///
+    /// You can set whether the console is allowed to enter the home menu with [Apt::set_home_allowed].
+    #[doc(alias = "aptIsHomeAllowed")]
+    pub fn is_home_allowed(&self) -> bool {
+        unsafe { ctru_sys::aptIsHomeAllowed() }
+    }
+
+    /// Immediately jumps to the home menu.
+    #[doc(alias = "aptIsHomeAllowed")]
```

Comment:
This alias looks like maybe a copy-paste mistake?

Reply:
oops